src/sys/socket/sockopt.rs

```diff
@@ -905,18 +905,31 @@ sockopt_impl!(
     libc::IP_PKTINFO,
     bool
 );
-#[cfg(any(linux_android, target_os = "freebsd", apple_targets, netbsdlike))]
+#[cfg(any(linux_android))]
```

Comment:
```suggestion
#[cfg(any(linux_android, bsd))]
```

DragonFlyBSD also has this, so `bsd` would work here